client/next-web/src/util/apiUtil.js

```diff
@@ -0,0 +1,9 @@
+export async function getDefaultCharacters() {
+  // TODO: use real hostname
+  const res = await fetch('http://127.0.0.1:8000/characters');
```

Comment:
Allow more urls to be served. We have a dev machine and a prod machine
https://github.com/Shaunwei/RealChar/blob/main/client/web/src/utils/urlUtils.js#L9